### DISCUSSION TITLE
Localization of javascript files

### DISCUSSION THREAD

**Discussion by kdinthehouse** [pixijs/pixijs#9955] [Q&A]
Is it possible to download these files and set that as source instead of importing it from cdn? because I'm hosting my site which is bandwidth restricted so it charges for overuse of content delivery network in order to eliminate that please tell me another way to import them instead of using cdn. please help

import * as PIXI from "https://cdn.skypack.dev/pixi.js@5.x";
import { KawaseBlurFilter } from "https://cdn.skypack.dev/@pixi/filter-kawase-blur@3.2.0";
import SimplexNoise from "https://cdn.skypack.dev/simplex-noise@3.0.0";
import hsl from "https://cdn.skypack.dev/hsl-to-hex";
import debounce from "https://cdn.skypack.dev/debounce";

**Comment by gabrielliwerant** [comment]
You can use node and npm: https://www.npmjs.com/. Once you're setup, e.g. `npm install pixi --save`
